detector/include/DCH_info.h

```diff
@@ -0,0 +1,341 @@
+#ifndef DCH_INFO_H_INCLUDED
+#define DCH_INFO_H_INCLUDED
+
+#include "TMath.h"
+
+#include "DDRec/DetectorData.h"
+#include <map>
+
+namespace dd4hep {  namespace rec {
+
+
+/* Data structure to store DCH excel table
```

Comment:
Why always the reference to the excel table?

Reply:
> I would even propose to attach it to the repo somehow if possible

Might be a good idea, I was confused where to find it after seeing it mentioned here so many times :)

Is the table simple enough to be exported to `.csv`? Otherwise another workaround can be to use libre office to convert it to `.fods` which is just the flat xml (non-compressed) version of `.ods`. Its not particularly nice but at least it can be meaningfully versioned with git...

Reply:
In case of saving the table in git, I would say ods/xls is the proper format so the formulas can be saved as well. The file size is 130kB

Reply:
I am sorry, the excel is internal to the IDEA-DriftChamber collaboration, and can not be public. I will remove all references to the excel table then